### PR TITLE
[Don't merge yet] blog post about wallabag v2

### PR DESCRIPTION
**Pull request by nicosomb**

### CODE REVIEW

File: content/20160403-wallabag-v2.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+date: 2016-04-03 21:00:00+00:00
+slug: wallabag-v2
+title: Three years after the first commit, wallabag 2.0.0
+tags: v2, release
+
+## Some history, some thanks
+
+[Three years ago, I started a little script, to replace the closed service I used to use](http://loeuillet.org/poche-pour-remplacer-instapaper-pocket-et-readability). It was a personal script, that I hosted on GitHub. I never realized that you could be interested with it.
```
Comment (tcitworld) [inline content/20160403-wallabag-v2.md]: Note that it's in french.

> I never realized that ~~you~~ **people** could be

File: content/20160403-wallabag-v2.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+![poche 0.1](/images/poche01.jpg)
+
+[You are more than 8.900 users on Framabag](https://framabag.org/analytics/), [you are more than 1.800 lovers on GitHub](https://github.com/wallabag/wallabag/stargazers), wallabag 1.x was downloaded more than 33.400 times, [we released so many versions](https://www.wallabag.org/pages/releases.html) with so many features, and our history can be still so long.
```
Comment (tcitworld) [inline content/20160403-wallabag-v2.md]: lovers --> fans

>  our history can be still so long

*I prefer* can still go on*, but you can keep it that way*

File: content/20160403-wallabag-v2.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+To all of our users: **Merci**.
+
+If our project is still alive today, it's also thanks to Thomas Citharel ([@tcitworld](https://twitter.com/tcitworld)). When I was not available for the project, he was always here to fix bugs or to reply to your questions. **Merci Thomas**!
+
+wallabag v2 is in my head for more than 2 years. But this version that we release today is also here thanks to Jérémy Benoist (@j0k): without his help on many technical subjects, wallabag v2 wouldn't exist. **Merci Jérémy**!
+
+Managing such a project like wallabag is so exciting (but it's also very tiring). Today, it's a big part of my personal life, that's why I also want to thank my wife, because it's not easy every day to share life with a wallaby. **Merci Audrey**!
```
Comment (tcitworld) [inline content/20160403-wallabag-v2.md]: > to share *your* life

File: content/20160403-wallabag-v2.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+Stop the bla bla!
```
Comment (tcitworld) [inline content/20160403-wallabag-v2.md]: "stop the blabbering" *could do*
*Maybe* "Enough with this" *would be better*

I really think english speakers don't know bla bla.

File: content/20160403-wallabag-v2.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+## How can you help us?
+
+By using wallabag, by reporting bugs, by translating wallabag and its documentation, by talking about wallabag to your friends, ...
+
+Don't forget that [you can also support us via Paypal, Flattr, Gratipay or Bountysource](https://www.wallabag.org/pages/donations.html).
+
+## What's next?
+
+~~Thanks to your donations, we can now take some holidays in the Bahamas, thank you so much.~~
```
Comment (tcitworld) [inline content/20160403-wallabag-v2.md]: Replace with Panama, it's quite fun at the moment.

Comment (nicosomb) [inline content/20160403-wallabag-v2.md]: Pas de souci, je vous emmène où vous voulez !

File: content/20160403-wallabag-v2.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+By using wallabag, by reporting bugs, by translating wallabag and its documentation, by talking about wallabag to your friends, ...
+
+Don't forget that [you can also support us via Paypal, Flattr, Gratipay or Bountysource](https://www.wallabag.org/pages/donations.html).
+
+## What's next?
+
+~~Thanks to your donations, we can now take some holidays in Panama, thank you so much.~~
+
+I'm sure you'll find bugs, I'm sure you'll have new ideas to implement... We already planned to release wallabag 2.1.0 at July, 3rd.
```
Comment (tcitworld) [inline content/20160403-wallabag-v2.md]: and obvious missing features :blush:

**Pull request merged**